src/FileResolver.php

```diff
@@ -16,14 +17,18 @@ class FileResolver
      */
     public function resolve(Configuration $configuration): array
     {
-        $finder = (new Finder())
-            ->in($configuration->getPaths())
-            ->name('*.php')
-            ->files()
-            ->followLinks()
-            ->ignoreUnreadableDirs(true)
-            ->ignoreVCS(true)
-            ->notPath($configuration->getExcludeFiles());
+        try {
+            $finder = (new Finder())
+                ->in($configuration->getPaths())
+                ->name('*.php')
+                ->files()
+                ->followLinks()
+                ->ignoreUnreadableDirs(true)
+                ->ignoreVCS(true)
+                ->notPath($configuration->getExcludeFiles());
+        } catch (DirectoryNotFoundException $exception) {
+            throw new \RuntimeException('Invalid directory provided for `paths`. '.$exception->getMessage());
```

Comment:
I don't think this message helps with the initial confusion. How about:

```suggestion
            throw new DirectoryNotFoundException($exception->getMessage().' Please make sure it is an existing, readable directory and not a file.');
```

This should result in the following message:
```
The "C:\users\p.makhnev\deptrac/1.php" directory does not exist. Please make sure it is an existing, readable directory and not a file.
```

Reply:
I think that this would be confusing as to why it has to exist. Where did this path come from? You would not know from the changed exception message.

The original message will point you to the exact place you have to change. To the configuration option that is wrong. 

Also, the message is generic enough to be applicable for other reasons why the directory might not exist. The patch being a file is not the only possibility.

Reply:
That sounds good to me. I think docs might be sufficient unless this will be a repeating occurrence. I that case I would reevaluate doing something in-code.